checker/src/test/java/org/checkerframework/checker/test/junit/FenumSwingTest.java

```diff
@@ -18,7 +18,8 @@ public FenumSwingTest(List<File> testFiles) {
                 testFiles,
                 org.checkerframework.checker.fenum.FenumChecker.class,
                 "fenum",
-                "-Aquals=org.checkerframework.checker.fenum.qual.SwingVerticalOrientation,org.checkerframework.checker.fenum.qual.SwingHorizontalOrientation,org.checkerframework.checker.fenum.qual.SwingBoxOrientation,org.checkerframework.checker.fenum.qual.SwingCompassDirection,org.checkerframework.checker.fenum.qual.SwingElementOrientation,org.checkerframework.checker.fenum.qual.SwingTextOrientation");
+                "-Aquals=org.checkerframework.checker.fenum.qual.SwingVerticalOrientation,org.checkerframework.checker.fenum.qual.SwingHorizontalOrientation,org.checkerframework.checker.fenum.qual.SwingBoxOrientation,org.checkerframework.checker.fenum.qual.SwingCompassDirection,org.checkerframework.checker.fenum.qual.SwingElementOrientation,org.checkerframework.checker.fenum.qual.SwingTextOrientation",
+                "-AignoreTargetLocation");
```

Comment:
Can't we adapt these qualifiers? Or add a comment that this depends on e.g. #340?

Reply:
You mean reformat these qualifiers? #340 you mean the issue? I didn't get the relation with this file and that issue?